### PR TITLE
User are reporting potential issues with using overlay2

### PR DESCRIPTION
**Pull request by rhatdan**
We should just switch to overlay right away rather then
handling the driver code lower in the stack.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: types/options.go [hunk 1]
```diff
@@ -190,6 +190,10 @@ func getRootlessStorageOpts(rootlessUID int, systemOpts StoreOptions) (StoreOpti
 	if driver := os.Getenv("STORAGE_DRIVER"); driver != "" {
 		opts.GraphDriverName = driver
 	}
+	// override users selection of 'overlay2' with 'overlay'
+	if opts.GraphDriverName == "overlay2" {
```
Comment (TomSweeneyRedHat) [inline types/options.go:194]: lint is barfing, it wants a constant for "overlay2"

**Comment by TomSweeneyRedHat** [comment]
@rhatdan looks like you need a rebase

File: types/options.go [hunk 2]
```diff
@@ -190,6 +194,10 @@ func getRootlessStorageOpts(rootlessUID int, systemOpts StoreOptions) (StoreOpti
 	if driver := os.Getenv("STORAGE_DRIVER"); driver != "" {
 		opts.GraphDriverName = driver
 	}
+	// override users selection of 'overlay2' with 'overlay'
```
Comment (TomSweeneyRedHat) [inline types/options.go:197]: Might want to leave behind a comment as to why the override is happening.

**Comment by rhatdan** [comment]
This is a Potentially breaking change, and should be held off until 4.0 of Podman.

**Review by vrothberg — COMMENTED** [review]
@rhatdan, I think it's time to get this one in.

**Pull request merged**